### PR TITLE
Et 1134 email from displays encoded html entities

### PR DESCRIPTION
**Pull request by redscar**
### 🎫 Ticket

[ET-1134] 

### 🗒️ Description

Single quotes were being escaped. Gmail did not like this. I removed the escaping that was occurring on the persons name

Bug Fix -

Single line change for TC sending emails for Paypal.

### 🎥 Artifacts 

https://user-images.githubusercontent.com/12241059/122456218-4e81f980-cf7b-11eb-86c5-d9eb7d700884.mp4

### ✔️ Checklist
- [x] I've included a changelog entry both in readme.txt and changelog.txt fi>
- [x] My code is tested. <!-- Check that tests are passing and DO NOT merge i>
- [x] My code has [proper inline documentation](https://the-events-calendar.g>

[ET-1134]: https://theeventscalendar.atlassian.net/browse/ET-1134

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 == Changelog ==
-
+* Fix - Prevent Tribe Commerce 'Confirmation email sender name' from displaying improperly when a single quote is added. [ET-1134]
```
Comment (juanfra) [inline changelog.txt:2]: The entry should go to `readme.txt` to the appropriate version. Then, once packaging before release, the release engineer (Gustavo) will take care of putting it in the right place. If you want, in any case, you can sync the readme entries here as part of your PR.

File: src/Tribe/Tickets.php [hunk 2]
```diff
@@ -2392,9 +2392,11 @@ public function send_tickets_email_for_attendee( $to, $tickets, $args = [] ) {
 
 				$headers[] = sprintf(
 					'From: %1$s <%2$s>',
-					filter_var( $from_name, FILTER_SANITIZE_STRING ),
+					stripcslashes( $from_name ),
 					$from_email
 				);
+				//html_entity_decode( $row[ $column_id ] , ENT_QUOTES | ENT_XML1, 'UTF-8');
+
```
Comment (juanfra) [inline src/Tribe/Tickets.php:2399]: This is commented code, I'm not sure if it was part of other WIP?

**Review by juanfra — CHANGES_REQUESTED** [review]
Nice! only a few changes. Please whenever you're working on a ticket be sure you're prefixing your branch with `fix/`, for example `fix/ET-1134-email-from-......`

File: changelog.txt (hunk 1 above)
Comment (redscar) [reply changelog.txt:2]: I just updated my branch and the readme.txt file has the notes in it as well

**Pull request closed without merging**